code/modules/events/aurora_caelus.dm

```diff
@@ -37,13 +37,13 @@
 		if(istype(affected_area, /area/station/service/kitchen))
 			for(var/turf/open/kitchen in affected_area)
 				kitchen.set_light(1, 0.75)
-			if(!prob(1) && !SSevents.holidays?[APRIL_FOOLS])
-				continue
+//			if(!prob(1) && !SSevents.holidays?[APRIL_FOOLS]) // SKYRAT EDIT BEGIN - Always in your kitchen
+//				continue
 			var/obj/machinery/oven/roast_ruiner = locate() in affected_area
 			if(roast_ruiner)
 				roast_ruiner.balloon_alert_to_viewers("oh egads!")
 				var/turf/ruined_roast = get_turf(roast_ruiner)
-				ruined_roast.atmos_spawn_air("plasma=100;TEMP=1000")
+				ruined_roast.atmos_spawn_air("hydrogen=100;TEMP=1000") // SKYRAT EDIT END - OG: plasma=100
```

Comment:
TG still might actually like this upstream because plasma fires from the oven are pretttyyyy intense for a joke. Making it a lsmall and easy to fight oven fire instead of a 'burn down the kitchen' fire would be nice

Reply:
This is in TGstation afaik, I just toned it down with a weaker gas and skipped the check.

Reply:
(I meant TG might like the weaker gas so they have an excuse to have the oven explode more often) ;)